UI/Web/src/app/_services/utility.service.ts

```diff
@@ -0,0 +1,23 @@
+let swipeCoord = [0, 0];
+let swipeTime = new Date().getTime();
+
+export function swipe(e: TouchEvent, when: 'start' | 'end'): ('right' | 'left') | undefined {
```

Comment:
You misunderstood me when I said put this on the Utility serivce. I expect this detectSwipeDirection would exist as a method on the Utility Service. Not an exposed function, but should be used and exposed like all other methods on the UtilityService.

Reply:
See bulk-selection Service  for an example of how services are used in regards to events like this.